app/src/main/java/com/github/palFinderTeam/palfinder/meetups/activities/MeetUpView.kt

```diff
@@ -181,7 +194,20 @@ class MeetUpView : AppCompatActivity() {
             viewModel.joinOrLeave(this)
         }
     }
-
+    /**
+     * cannot mute/unmute a meetUp if you are not logged in
+     */
+    fun onMuteOrUnmute(v: View){
+        if(profileService.getLoggedInUserID() == null){
+            createPopUp(this,
+                { startActivity(Intent(this, LoginActivity::class.java)) },
```

Comment:
Maybe we should do a helper for these recurrent popups (also if you want to test this don't hesitate to see how I did with the warning when joining a meetup with a blocked user)